src/unitxt/processors.py

```diff
@@ -75,6 +75,15 @@ def process(self, instance):
         return splitted[0].strip()
 
 
+class ConvertToBoolean(BaseFieldOperator):
+    def process(self, instance):
+        if any(w in str(instance).strip().lower() for w in ["yes", "right", "correct"]):
```

Comment:
small thing: i suggest to assign `str(instance).strip().lower()` once into some variable and then use that in the tests. to avoid that being done 6 times (in the worst case)..

Reply:
done